Marlin/src/HAL/ESP32/u8g_esp32_spi.cpp

```diff
@@ -0,0 +1,88 @@
+#ifdef ARDUINO_ARCH_ESP32
+
+#include "../../inc/MarlinConfigPre.h"
+#ifdef MKS_MINI_12864_V3
```

Comment:
This should be `FYSETC_MINI_12864_2_1` so it works with the other LCD aliases:

https://github.com/MarlinFirmware/Marlin/blob/db4172b5fa05e818f42427001398db5f13d13024/Marlin/src/inc/Conditionals_LCD.h#L34-L37

Reply:
I really wish these alias where gone they don't add anything and make life more complicated. But added